code/modules/mob/living/basic/bots/cleanbot/cleanbot_abilities.dm

```diff
@@ -0,0 +1,36 @@
+/datum/action/cooldown/mob_cooldown/bot
+	background_icon_state = "bg_revenant"
+	overlay_icon_state = "bg_revenant_border"
```

Comment:
Why revenant over tech or tech_blue?
No judgement if you think it looks better, just would have figured those were appropriate for a bot.